e2e/Makefile

```diff
@@ -41,12 +44,7 @@ KIND_CONFIG="topolvm-cluster-single-writer.yaml"
 MINIKUBE_FEATURE_GATES="ReadWriteOncePod=true"
 endif
 
-ifeq ($(STORAGE_CAPACITY),true)
 HELM_VALUES_FILE_LVMD := manifests/values/daemonset-lvmd-storage-capacity.yaml
-HELM_VALUES_FILE := manifests/values/storage-capacity.yaml
-else
-HELM_VALUES_FILE_LVMD := manifests/values/daemonset-lvmd.yaml
```

Comment:
I could not understand why `manifests/values/daemonset-lvmd.yaml` is no longer needed. (Now `HELM_VALUES_FILE_LVMD` is always set to `manifests/values/daemonset-lvmd-storage-capacity.yaml`.)
Could you tell me your aim?

Reply:
daemonset-lvmd variants are used only by the minikube environment, and I dropped support of running topolvm-scheduler on minikube in [this commit](https://github.com/topolvm/topolvm/pull/490/commits/43d8e7c36e812023a2b46ddb04e338e6673373c9). Therefore, nobody use `daemonset-lvmd.yaml` is now.

Reply:
P.S. daemonset-lvmd enables topolvm-scheduler in default.